igdiscover/init.py

```diff
@@ -29,6 +29,8 @@
 def add_arguments(parser):
 	parser.add_argument('--database', '--db', metavar='PATH', default=None,
 		help='Directory with V.fasta, D.fasta and J.fasta files. If not given, a dialog is shown.')
+	parser.add_argument('--species', '--sp', metavar='SPECIES', default='any',
```

Comment:
I tried removing '--db' which was there before and when I run
```
igdiscover init --help
...
  --database PATH       Directory with V.fasta, D.fasta and J.fasta files. If
                        not given, a dialog is shown.
```
Removing '--sp' would be inconsistent with '--db'. I suggest keeping them, if you meant that.
User might not know if it can be abbreviated with a short form, otherwise.

Reply:
`--sp` isn’t too much of an improvement over `--species`, so I don’t see the advantage. `--db` is not a prefix of `--database`, which is why it has to be specified explicitly. "db" is an often used abbreviation for database.

If anything, `--db` would be worth removing, but that’s not backwards-compatible and just not worth it.